sample/src/main/java/net/grandcentrix/thirtyinch/sample/HelloWorldPresenter.java

```diff
@@ -44,13 +45,13 @@
     protected void onAttachView(@NonNull final HelloWorldView view) {
         super.onAttachView(view);
 
-        rxSubscriptionHelper.manageViewSubscription(mText.asObservable()
-                .subscribe(view::showText));
-
-        rxSubscriptionHelper.manageViewSubscription(view.onButtonClicked()
+        final Subscription showTextSub = mText.asObservable().subscribe(view::showText);
+        final Subscription onButtonClickSub = view.onButtonClicked()
                 .subscribe(aVoid -> {
                     triggerHeavyCalculation.onNext(null);
-                }));
+                });
+
+        rxSubscriptionHelper.manageViewSubscriptions(showTextSub, onButtonClickSub);
```

Comment:
I'm not sure if it's useful to create local variables for the subscriptions. But I also don't like method references. Just personal preference here 🦄